constraint/pkg/apis/templates/v1/scheme.go

```diff
@@ -0,0 +1,42 @@
+package v1
+
+import (
+	ctschema "github.com/open-policy-agent/frameworks/constraint/pkg/schema"
+	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions"
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	"k8s.io/apiextensions-apiserver/pkg/apiserver/schema"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+const version = "v1"
+
+var (
+	structuralSchema *schema.Structural
+	sch              *runtime.Scheme
+)
+
+func init() {
+	// Prevent problems with ordering of init() function calls.  These
+	// functions are called according to the lexicographic order of their
+	// containing files.  As Register() is called on the localSchemeBuilder by
+	// zz_generated.conversion.go, the conversion functions haven't been
+	// registered with the localSchemeBuilder by the time this init() function
+	// runs.  We sidestep this problem by adding RegisterConversions here.
+	sb := runtime.NewSchemeBuilder(SchemeBuilder.AddToScheme, addDefaultingFuncs)
```

Comment:
@maxsmythe @willbeason 

While I got rid of `localSchemeBuilder...`, I'm still using `SchemeBuilder.AddToScheme`.  This is a package level variable created in [register.go](https://github.com/open-policy-agent/frameworks/blob/master/constraint/pkg/apis/templates/v1/register.go#L37).  I could replicate the code there within this `init()` function, or as package level variables in that file.  But, I didn't see a real upside to duplicating that code as the usage I have above already decouples me from other `init()` functions, which was the real problem.

LMK if you disagree.

Reply:
Looks good.